subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/GroovyGradlePluginInitIntegrationTest.groovy

```diff
@@ -18,11 +18,13 @@ package org.gradle.buildinit.plugins
 
 import org.gradle.buildinit.plugins.fixtures.ScriptDslFixture
 import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import org.gradle.util.internal.VersionNumber
 import spock.lang.IgnoreIf
 import spock.lang.Issue
 
 import static org.gradle.buildinit.plugins.internal.modifiers.BuildInitDsl.GROOVY
 
+@IgnoreIf({ VersionNumber.parse(GroovySystem.version).major >= 4}) // FIXME if Groovy 4 is bundled, cannot run without regenerating library-versions.properties
```

Comment:
Given the problem Octavia found with a test which was ignored for 8 years, do we have plans to revisit this?